transfer/metadata.md

```diff
@@ -2,141 +2,6 @@
 
 A file named `metadata.json` should be included in the `data/` directory of the bag.
 
-```
-{
-  "identifier": "OyGpXmSFVkCpds7i4gRv",
-  "title": "Annual Reports",
-  "dates":[
-    {
-      "start": "1994-01-01",
-      "end": "1997-01-01"
-    }
-  ],
-  "creators": [
-    {
-      "name": "Ford Foundation",
-      "id": "https://www.fordfoundation.org#organization",
-      "role": "creator",
-      "type": "organization"
-    }
-  ],
-  "languages": ["http://id.loc.gov/vocabulary/iso639-2/eng", "http://id.loc.gov/vocabulary/iso639-2/spa"],
-  "description": "Annual reports discussing the accomplishments and major strategic initiatives of the Ford Foundation.",
-  "restrictions": [{}]
-}
-```
+This file must against either [JSON]("http://www.json.org/") or [JSON-LD]("http://json-ld.org/") formats. All of our donors are welcome to add any serialized metadata that they would like to provide the RAC in this file.
```

Comment:
Let's change the structure of this a bit:
Metadata is in two places
1. bag-info.txt (point to bagit spec) - this is what allows us to manage records in the bag archivally.
2. in a JSON file (discuss below. - this should include everything donors want to say about the records in the bag.